src/database/seeds/relational/user/user-seed.service.ts

```diff
@@ -54,13 +59,25 @@ export class UserSeedService {
         } as UserEntity),
       );
     }
+
```

Comment:
@stanislavkhatko  left a comment:

> There is user.role.permissions relation, not user.permissions. I believe userPermission was implemented to extend user role permissions with extra possibilities. This should be done with number of roles instead.

I haven't played much in this section yet. Can you elaborate?  This looks like it gives the user all permissions.  The user is either the test or admin user.   Are you saying that we should just assign a role with all permissions?

Reply:
@tompscanlan we about to have a naming conflict with things, but `user` has manyToMany with `role` witch has relation to 'PermissionEntity', that should be expected nested check by fetching `user.role.permissions` relation in the guard. Instead there is `user.userPermissions` witch connects user with permissions without `role`. I was laying on nesting and more appealing approach of `user.role.permissions`, can't find the benefit of having user.userPermissions for now.

Reply:
We're now using the role to look up permissions.  On the user, I've kept the user permissions, but it can come out in another commit.  I append the user role permissions to the user permissions when we check user permissions.  Should make it easy to remove, but available if we do need it.